CorsixTH/Lua/dialogs/fullscreen/annual_report.lua

```diff
@@ -452,7 +452,6 @@ end
 function UIAnnualReport:close()
   if TheApp.world:getLocalPlayerHospital().game_won then
```

Comment:
It seems odd we check for game won before we actually run the function in world for `checkIfGameWon()` but that isn't relevant here.

Reply:
I would say, the problem is the distributed control of the blue filter.

But yeah, not relevant for this issue.